pkg/reconciler/shared/tektonconfig/instance.go

```diff
@@ -120,6 +120,7 @@ func (tc tektonConfig) createInstance() error {
 			},
 		},
 	}
+	tcCR.SetDefaults(context.TODO())
```

Comment:
I think this should use `ctx`, meaning `createInstance` should take a `context.Context` as argument, otherwise we colud miss actual configured defaults

Reply:
also there is [line](https://github.com/tektoncd/operator/pull/415/files/9a80ec5b834597476f58b4c01477d0e3fac304b1#diff-5559ff56481d7f8604560c6ceb814063d2b6873416abaadb45e63bc07d1f9db8R125) where context.TODO is used. Shall I change that also?

Reply:
I don't see the line but.. maybe, yeah ?

Reply:
maybe it was moved with my commit. I have changed that with the `ctx` [here](https://github.com/tektoncd/operator/pull/415/files#diff-5559ff56481d7f8604560c6ceb814063d2b6873416abaadb45e63bc07d1f9db8L124)